tests/overhead/dockerfiles/start_app.sh

```diff
@@ -2,7 +2,7 @@
 
 composer --working-dir=/var/www install
 
-if [[ -z "${DD_TRACE_LIBRARY_VERSION}" ]]; then
+if [[ ! -z "${DD_TRACE_LIBRARY_VERSION}" ]]; then
```

Comment:
Was this change intentionally committed?

Reply:
This is the actual fix. We did not use very much compare to specific version and it was overseen. The logic is: "if it is not empty then ---> install"